roles/feature-rhoai/tasks/post-deploy.yaml

```diff
@@ -3,45 +3,23 @@
 # https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.16/html/installing_and_uninstalling_openshift_ai_self-managed/installing-and-deploying-openshift-ai_install#installing-openshift-ai-operator-using-cli_operator-install
 
 # TODO: Do we need to create Object Storage on ISAR ODF???
-
-
-- name: Get child cluster access
-  community.hashi_vault.vault_kv2_get:
-    url: "{{ lookup('ansible.builtin.env', 'RH_VAULT_URL' ) }}"
-    auth_method: approle
-    role_id: "{{ lookup('ansible.builtin.env', 'RH_VAULT_ROLE_ID' ) }}"
-    secret_id: "{{ lookup('ansible.builtin.env', 'RH_VAULT_SECRET_ID' ) }}"
-    ca_cert: "{{ lookup('ansible.builtin.env', 'RH_VAULT_CA_CERT_FILENAME' ) }}"
-    engine_mount_point: apps
-
-    path: "coe-lab/cluster-credential/stormshift-{{ inventory_hostname }}"
-  register: cluster_credential
-
-- name: Temp folder
-  ansible.builtin.tempfile:
-    state: "directory"
-  register: temp
-  changed_when: False
-
-- name: Write kubeconfig
-  ansible.builtin.copy:
-    dest: "{{ temp.path }}/kubeconfig"
-    content: "{{ cluster_credential.secret.kubeconfig }}"
-  changed_when: False
-
```

Comment:
@DanielFroehlich mh maybe I'm blind, but it looks like the
```
- name: Fetch kubeconfig from vault
  ansible.builtin.include_role:
    name: internal-fetch-kubeconfig
``` 
is missing?

Reply:
That is in line 13 below